File: examples/museum/oseg_generated/java/GetSpecialEventDefaultExample.java
package org.openapimuseum.client.examples;

import org.openapimuseum.client.ApiException;
import org.openapimuseum.client.Configuration;
import org.openapimuseum.client.api.*;
import org.openapimuseum.client.auth.*;
import org.openapimuseum.client.model.*;

import java.io.File;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.List;
import java.util.Map;

public class GetSpecialEvent_default_example
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();

        try
        {
            var response = new EventsApi(config).getSpecialEvent(
                "dad4bce8-f5cb-4078-a211-995864315e39"
            );

            System.out.println(response);
        } catch (ApiException e) {
            System.err.println("Exception when calling Events#getSpecialEvent");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: examples/museum/oseg_generated/ruby/GetSpecialEventDefaultExample.rb
require "openapimuseum_client"

OpenApiMuseumClient.configure do |config|
end

begin
    response = OpenApiMuseumClient::EventsApi.new.get_special_event(
        "dad4bce8-f5cb-4078-a211-995864315e39",
    )

    p response
rescue OpenApiMuseumClient::ApiError => e
    puts "Exception when calling Events#get_special_event: #{e}"
end
